plugins/Login/Controller.php

```diff
@@ -163,6 +162,26 @@ function logme()
     }
 
     /**
+     * Error message shown when an AJAX request has no access
+     *
+     * @param string $errorMessage
+     * @return string
+     */
+    public function ajaxNoAccess($errorMessage)
+    {
+        return sprintf(
+            '<div class="alert alert-danger">
+                <p><strong>%s:</strong> %s</p>
+                <p><a href="%s">%s</a></p>
+            </div>',
+            Piwik::translate('General_Error'),
+            $errorMessage,
+            'index.php?module=Login&idSite=7&period=day&date=yesterday',
```

Comment:
idSite is hardcoded to 7

In general Login URL needs to be only `index.php?module=Login` (other params not needed)